packages/lesswrong/lib/scoring.ts

```diff
@@ -31,7 +31,7 @@ export const TIME_DECAY_FACTOR = timeDecayFactorSetting;
 export const FRONTPAGE_BONUS = frontpageBonusSetting;
 export const CURATED_BONUS = curatedBonusSetting;
 
-const getSubforumScoreBoost = (): SubforumCommentBonus => {
+export const getSubforumScoreBoost = (): SubforumCommentBonus => {
```

Comment:
I think this function is slightly misnamed? Isn't in being used for all comments? (I could be dramatically misreading)

Reply:
Broadly pretty confused about this function

Reply:
Nope - it's only used for subforum comments. We filter for `"tagCommentType" = 'SUBFORUM'` in Postgres at line 201 of `updateScores.ts` and in Mongo at line 140 of `scoring.ts`.